Library/Formula/xsd.rb

```diff
@@ -0,0 +1,49 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
```

Comment:
You can keep your file locally and install directly from it: `brew install ./xsd.rb`.